Exec/SCIENCE/rotating_star/tagging.F90

```diff
@@ -58,4 +62,33 @@ subroutine state_error(tag,tag_lo,tag_hi, &
 
   end subroutine state_error
 
+  subroutine tag_boxes(tag,tag_lo,tag_hi, &
```

Comment:
Again, this is partially because I was switching branches and hadn't noticed, but the tagging module for this problem needed updating so that this problem would compile.